Session04/로그인회원가입과제/src/components/TextField.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import styled from "styled-components";
+
+const TextField = (textFieldProps) => {
+  const { label, placeholder, value, onChange } = textFieldProps;
+  return (
+    <TextFieldContainer>
+      <label>{label}</label>
+      <input
```

Comment:
label과 input에는 for와 id를 연결해 보세요!
이유는 사용자의 편의성을 위해서요 :)

구글링 해보시면 쉽게 적용하실 수 있을 거예요!
(HTML에서와 React에서 속성 이름이 살짝 다를 거예요)

Reply:
```suggestion
      <label>  <input type="text" 
        placeholder={placeholder}
        value={value}
        onChange={onChange}
      />
{label}</label>
```
    
또는 이런식으로 label 안에 input을 집어 넣어서 연결해주는 방법도 있답니당